README.md

```diff
@@ -1,141 +1,54 @@
-Whenever a module is accepted by Lifecycle Manager the ModuleTemplate gets translated into a Manifest, which describes the actual desired state of the module operator.
+### Prerequisites
 
-The Lifecycle Manager then updates the [Kyma Custom Resource](api/v1alpha1/kyma_types.go) of the cluster based on the observed status changes in the Module Custom Resources (similar to a native kubernetes deployment tracking availability).
+To use Lifecycle Manager in a local setup, install the following:
 
-Module operators only have to watch their own custom resources and reconcile modules in the target clusters to the desired state. 
+- [k3d](https://k3d.io/)
+- [istioctl](https://istio.io/latest/docs/setup/install/istioctl/)
+- [Kyma CLI](https://kyma-project.io/docs/kyma/latest/04-operation-guides/operations/01-install-kyma-CLI)
 
-### Example
+### Steps
 
-A sample `Kyma` CR could look like this:
-```
-apiVersion: operator.kyma-project.io/v1beta1
-kind: Kyma
-metadata:
-  name: my-kyma
-spec:
-  modules:
-  - name: my-module
-```
+1. To set up the environment, provision a local k3d cluster and install Kyma. Run:
 
-The creation of the custom resource triggers a reconciliation that
-1. looks for a ModuleTemplate based on search criteria, for example the OCM Component Name of the Module or simply the of the `ModuleTemplate`
-2. creates a `Manifest` for `my-module` based on a [ModuleTemplate](api/v1beta1/moduletemplate_types.go) found in the cluster by resolving all relevant image layer for the installation
-3. installing the contents of the modules operator by applying them to the cluster, and observing its state
-4. reporting back all states observed in the `Manifest` which then gets propagated to the `Kyma` resource for the cluster.
-   Lifecycle Manager then uses this to aggregate and combine the readiness condition of the cluster and determine the installation state or trigger more reconciliation loops as needed.
+  ```bash
+  kyma provision k3d
+  kyma alpha deploy
+  ```
 
-As mentioned above, when each module operator completes their installation, it reports its own resource status. However, to accurately report state, we read out the `.status.state` field to accumulate status reporting for an entire cluster.
+2. Apply a module template. Run the following kubectl command:
 
-### Getting Started
+  ```bash
+  kubectl apply -f {MODULE_TEMPLATE.yaml}
+  ```
 
-If you are new to our Lifecycle Manager and want to get started quickly, we recommend that you follow our [Quick Start Guide](./docs/user/quick-start.md). This guide will walk you through the basic steps of setting up your local KCP cluster to installing the Lifecycle Manager, and using the main features.
+3. Enable a module. Run:
 
-## Architecture
+  ```bash
+  kyma alpha enable module {MODULE_NAME}
```

Comment:
Depending on the ModuleTemplate the user may need to add channel/namespace/kyma name

Reply:
I forgot about that but as far as I remember now, we should not mention channels in the OS documentation, is that correct?